src/python/pants/backend/python/tasks2/python_task_mixin.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.backend.python.tasks2.gather_sources import GatherSources
+from pants.backend.python.tasks2.partition_targets import PartitionTargets
+from pants.backend.python.tasks2.select_interpreter import SelectInterpreter
+
+
+class PythonTaskMixin(object):
```

Comment:
Does this have to be a mixin? I think it can be a subclass of `Task`, that `ResolveRequirementsTaskBase` can extend?